packages/core/src/contrib/Queries.ts

```diff
@@ -58,3 +59,27 @@ export const polygonLikePoints = ([t, s]: [string, any]): ad.Pt2[] => {
     throw new Error(`${t} not supported for polygonLikePoints.`);
   }
 };
+
+/**
+ * Return -1.0 for negative number, +1.0 otherwise.
+ */
+const signOf = (x: ad.Num): ad.Num => {
+  const negative = lt(x, 0);
+  return ifCond(negative, -1, 1);
+};
```

Comment:
Can't/shouldn't you instead just use `sign` from `AutodiffFunctions`?

Reply:
Nice catch, didn't notice that. Also removed `msign` from `Functions.ts`

Reply:
Cool, thanks! @joshsunshine just to check: is this OK? I vaguely remember having a conversation previously about how `msign` is specifically meant to be distinct from `sign` in that it differs at zero. Should we add it back with a documentation comment clarifying this distinction, or is it OK to remove it?

Reply:
@jiriminarcik I don't think we should remove `msign` for the reason @samestep describes above.

Reply:
I see.. Don't worry, it's back! Thanks for clarification. :)